examples/serverless/pytorch/data_ingestion.py

```diff
@@ -0,0 +1,71 @@
+import os
+from typing import Any
+
+import numpy as np
+import tiledb.cloud
+
+# Your TileDB username and password, exported as environmental variables
+TILEDB_USER_NAME = os.environ.get("TILEDB_USER_NAME")
+TILEDB_PASSWD = os.environ.get("TILEDB_PASSWD")
+
+# Your TileDB namespace
+TILEDB_NAMESPACE = "your_tiledb_namespace"
+
+# Your S3 bucket
+S3_BUCKET = "your_s3_bucket"
+
+IMAGES_URI = "tiledb://{}/s3://{}/mnist_images".format(TILEDB_NAMESPACE, S3_BUCKET)
+LABELS_URI = "tiledb://{}/s3://{}/mnist_labels".format(TILEDB_NAMESPACE, S3_BUCKET)
+
+
+# Let's define an ingestion function
+def ingest_in_tiledb(data: np.array, batch_size: int, uri: str) -> None:
```

Comment:
Could we use inside this function the already implemented `ingest_in_tiledb` function. A general comment is that we should maybe develop an `ingestors` package if we can generalise the ingestion since it is our most used function in general 😄